pldoc/_components/breadcrumbs.md

```diff
@@ -0,0 +1,30 @@
+---
+layout:                 pattern
+title:                  Breadcrumbs
+date:                   2016-02-29 00:00:00
+
+categories:             component
+tags:
+- atomic
+- pattern
+- js
+- breadcrumbs
+- nav
+
+slug:                   breadcrumbs
+
+url_styles:             "patterns/_breadcrumbs.scss"
+
+description:            Breadcrumb navigation.
+
+info:                   Accessible tiered navigation using breadcrumbs.
+ui-toolkit-component:   breadcrumbs
+js:                     "/examples/breadcrumbs-js.html"
+---
+<div class="breadcrumbs-example">
```

Comment:
Can you add a pre-rendered version too (like @AlasdairSwan does with the dropdown menu). I think it will be useful to be able to render breadcrumbs server-side sometimes.